CMakeLists.txt

```diff
@@ -49,7 +49,8 @@ elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang" OR "${CMAKE_CXX_COMPILER_ID}
 endif()
 
 if ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "AppleClang")
-  set(CMAKE_VISIBILITY_INLINES_HIDDEN ON)
+  set(CMAKE_CXX_VISIBILITY_PRESET hidden)
+  set(CMAKE_VISIBILITY_INLINES_HIDDEN 1)
```

Comment:
What warnings did visibility rules cause?

Reply:
ON and 1 are the same.  I was trying to make it consistent with other places in our source code.